README.adoc

```diff
@@ -1,17 +1,223 @@
-= Android uLink
+= uProtocol Client Android Java Library
 :toc:
+:toclevels: 4
+:sectnums:
+:source-highlighter: coderay
 
 == Overview
+The following is the uProtocol client library that implements uTransport, RpcClient and RpcService defined in https://github.com/eclipse-uprotocol/up-java[uProtocol Java SKD] using Android Binder. It also includes some commonly used utilities for error handling.
 
-the following is the Java Android (Binder) ulink library that implements uTransport and RpcClient defined in https://github.com/eclipse-uprotocol/uprotocol-spec/tree/main[uProtocol Specifications]. 
+== Getting Started
+=== Importing the Library
+If you are using Gradle, add the following to your _build.gradle_ file's dependencies:
 
+[,groovy]
+----
+android {
+    dependencies {
+        implementation 'org.eclipse.uprotocol:up-client-android-java::1.5.+'
+    }
+}
+----
 
-== Getting Started
+=== Configuring the Library
+`ULink`, by default, establishes a connection to uBus service that is integrated into the system as part of `"org.eclipse.uprotocol.core"` package.
```

Comment:
ulink -> upclient and it will be in org.eclipse.uprotocol package like we talked about

Reply:
done